### ISSUE TITLE
modelSystem

### ISSUE THREAD

**Issue by anngvu** [nf-osi/nf-metadata-dictionary#9]
This concept is currently being used out in the wild but is not found in the schema. It’s possible that what should be used instead is modelSystemName. 

Recommendation: ?

**Comment by allaway** [comment]
I agree, I think this is a typo for modelSystemName, but can you share some example values? 
Similar to #42, I'm not sure if we should remove the modelSystem annotation, in case someone has code that relies on it. What do you think?
